src/main/markdown/examples.md

```diff
@@ -155,6 +155,16 @@ GWT is being used by tens of thousands of projects around the world. Take a look
 
 <div class="project">
   <div class="screenshot">
+    <a href="https://github.com/gchq/stroom" target="_blank"><img src="https://github.com/gchq/stroom-docs/raw/master/resources/screen-shot-feed-tn.png" alt="ruxit"/></a>
```

Comment:
`alt` should say Stroom, not Ruxit. Also, maybe link to a specific commit instead of `master` (in case they move the file around in a later commit)